src/components/resources/Resources.js

```diff
@@ -118,27 +133,57 @@ const Resources = () => {
                     onButtonClick={loadGetResources}
                     onResetSearch={handleRefreshSearch}
                 />
+
+                <br />
+                <Typography
+                    style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+                >
+                    Filters
+                </Typography>
+                <DropdownFilter
+                    placeholder="Select a Media Type Tag"
```

Comment:
Can you make this placeholder text match that in the search bar? I think it will make the top of the page look more cohesive. Ie shrink and make gray